src/main/java/racing/model/CarPosition.java

```diff
@@ -0,0 +1,23 @@
+package racing.model;
+
+import racing.strategy.DefaultCarMoveStrategy;
+
+public class CarPosition {
+    private int position;
+
+    public CarPosition() {
+        this.position = 1;
```

Comment:
```suggestion
        this(1);
```

점진적 생성자 패턴을 활용하면 좋을 것 같네요